sections/featured-product.liquid

```diff
@@ -652,6 +652,34 @@
       "default": false,
       "label": "t:sections.featured-product.settings.enable_video_looping.label"
     },
+    {
+      "type": "select",
+      "id": "color_scheme",
+      "options": [
+        {
+          "value": "accent-1",
+          "label": "t:sections.all.colors.accent_1.label"
+        },
+        {
+          "value": "accent-2",
+          "label": "t:sections.all.colors.accent_2.label"
+        },
+        {
+          "value": "background-1",
+          "label": "t:sections.all.colors.background_1.label"
+        },
+        {
+          "value": "background-2",
+          "label": "t:sections.all.colors.background_2.label"
+        },
+        {
+          "value": "inverse",
+          "label": "t:sections.all.colors.inverse.label"
+        }
+      ],
+      "default": "background-1",
+      "label": "t:sections.all.colors.label"
+    },
```

Comment:
Based on the issue it looks like we also need to add a container color scheme settings. The figma has this: 
![](https://screenshot.click/22-01-304w2-9uszq.png)

Reply:
Also think it should be moved outside of the media settings: 
![](https://screenshot.click/22-01-8f5aa-gamty.png)

Reply:
Makes sense! 👍